osu.Game/Localisation/UserInterfaceStrings.cs

```diff
@@ -109,6 +109,11 @@ public static class UserInterfaceStrings
         /// </summary>
         public static LocalisableString ModSelectHotkeyStyle => new TranslatableString(getKey(@"mod_select_hotkey_style"), @"Mod select hotkey style");
 
+        /// <summary>
+        /// "Song select background blur"
+        /// </summary>
+        public static LocalisableString SongSelectBackgroundBlurLevel => new TranslatableString(getKey(@"song_select_background_blur_level"), @"Song select background blur");
```

Comment:
The setting is already under the song select header, so doesn't need the "song select' part.